Source/ITab_Lock.cs

```diff
@@ -222,5 +221,36 @@ private void OwnerCheckbox(Rect rect, Pawn pawn)
             GUI.DrawTexture(position, image);
             Text.Anchor = anchor;
         }
+
+        private static void CheckboxLabeled(Listing_Standard listing, ref bool anythingChanged, string label, ref bool checkOn, string tooltip = null, float height = 0.0f, float labelPct = 1f)
+        {
+            var current = checkOn;
+            listing.CheckboxLabeled(label, ref checkOn, tooltip, height, labelPct);
+            if (current != checkOn)
+                anythingChanged = true;
+        }
+
+        [SyncMethod]
+        private static void SetWantedStateData(ThingWithComps door, LockState newState)
+        {
+            var data = LockUtility.GetData(door);
+
+            if (newState.owners == null)
+                newState.owners = data.WantedState.owners;
```

Comment:
for sure you want to copy reference to list not creating a new one? Because assigning from clipboard may cause to rewrite older references

Reply:
My bad, going to fix this in a moment